bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/VBFToXToHH_UL/VBFToRadionToHH_M/VBFToRadionToHH_M_proc_card.dat

```diff
@@ -0,0 +1,12 @@
+
+set group_subprocesses Auto
+set ignore_six_quark_processes False
+set loop_optimized_output True
+set complex_mass_scheme False
+define vl = ve vm vt
+define vl~ = ve~ vm~ vt~
+import model heft_radion
+define q = u c d s b u~ c~ d~ s~ b~
+define p = u c d u~ c~ d~
```

Comment:
I am just taking a super quick look.
But is 3 flavor scheme for proton definition what you really want?

Reply:
Hello @sihyunjeon - Yes this is intentional. Our reasoning is that the proton pdf for quarks other than u/d is very negligible, to a level only precision analyses in H couplings need. Since we are producing X->HH, we don't think this level of precision is necessary, and we considerably decrease the number of diagrams. 

-Abe

Reply:
Hello @sihyunjeon - A friendly ping/reminder. What do you think about this?

Thank you,
Abe

Reply:
The number of diagrams will only affect the first step of sample production (wmLHEGEN) but this won't really make much difference in your full production as the most time consuming part is the SIM step where you run GEANT4 ==> It won't really make that much difference in time consumption (might help very little).
So I don't really disagree to your choice, it is valid point, but still it won't help you that much.

@agrohsje might give you answers on what to do.

Reply:
updated to 5F proton definition